src/fidesops/task/filter_results.py

```diff
@@ -0,0 +1,83 @@
+import logging
+from typing import List, Dict, Any, Union
+
+from fidesops.graph.config import FieldPath
+
+logger = logging.getLogger(__name__)
+
+
+def select_and_save_field(saved: Any, row: Any, target_path: FieldPath) -> Dict:
+    """Extract the data located along the given `target_path` from the row and add to the "saved" dictionary.
+
+    Entire rows are returned from your collections; this function will incrementally just pull the PII from the rows,
+    by retrieving data along target_paths to relevant data categories.
+
+    To use, you might start with a "saved" that is an empty dict, and loop through a list of FieldPaths you want,
```

Comment:
> you might start with a "saved"

Do you mean to say "saved" collection?

Reply:
Ha I meant the variable is called "saved", just like I refer to the "row" or the "target_path", but yes this is confusing, I'll rephrase.